pact/mailbox/mailbox.pact

```diff
@@ -157,35 +157,25 @@
    )
 
    (defun define-hook:string (hook:module{hook-iface})
-      (insert dependencies "default"
-         {
-            "hook": hook
-         }
-      )
-   )
+      (with-capability (ONLY_ADMIN)

```

Comment:
I find it a bit unfortunate to have all of these whitespace-only, non-semantic changes also in this PR. It would be nice to know that the diff contains ONLY changes related to fixing the Mailbox issues.